src/fal/run_scripts.py

```diff
@@ -72,8 +73,10 @@ def run_global_scripts(list: List[FalScript], project: FalProject):
             faldbt.source,
             faldbt.write_to_source,
             faldbt.write_to_firestore,
+            faldbt._manifest.get_models,
```

Comment:
it could have been `faldbt.list_models` too. Any reason why this one?

Reply:
that one includes status information

Reply:
ugh this just the names of the models though

Reply:
looks like its not in the latest main